src/Compilers/CSharp/Portable/Binder/Binder_Lambda.cs

```diff
@@ -327,10 +327,12 @@ private UnboundLambda BindAnonymousFunction(AnonymousFunctionExpressionSyntax sy
                         firstDefault = i;
                     }
 
+                    ParameterHelpers.GetModifiers(paramSyntax.Modifiers, out _, out var paramsKeyword, out _, out _);
```

Comment:
Please use named parameters for the discards.

Reply:
Thanks, added: 092b3b9e37fc4f4073277aba9ddc01127382da4d